src/frontend/vite.config.ts

```diff
@@ -11,11 +11,7 @@ export default defineConfig({
     },
     server: {
         proxy: {
-            '/api': {
-                target: 'https://localhost:7230',
-                changeOrigin: false,
-                rewrite: path => path.replace(/^\/api/, '')
-            }
+            '/api': 'http://localhost:5230'
```

Comment:
this shorthand is exactly what we need

Reply:
For now, we also redirect to the HTTP (**not HTTPS**) backend API. Later, we can update the config for HTTPS in prod environments